### DISCUSSION TITLE
Commits doesn't show on my contribution

### DISCUSSION THREAD

**Discussion by Aliyan-Ansari** [community/community#61350] [Projects and Issues]
### Select Topic Area

Question

### Feature Area

Issues

### Body

I have push the code in a repository but github doesn't show it in my cotribution

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by UmerGillani36** [comment]
Go to right top of the progress table and enable the private contribution
